config/metac.yaml

```diff
@@ -101,15 +101,6 @@ spec:
           values: ["cstorpoolclusters.openebs.io", "csinodeinfos.csi.storage.k8s.io", "csivolumes.openebs.io",
                     "volumesnapshotclasses.snapshot.storage.k8s.io", "volumesnapshotcontents.snapshot.storage.k8s.io",
                     "volumesnapshots.snapshot.storage.k8s.io"]
-  - apiVersion: storage.k8s.io/v1beta1
-    resource: csidrivers
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchReferenceExpressions:
-        - key: metadata.annotations.dao\.mayadata\.io/openebs-uid
-          refKey: metadata.uid # match this ann value against watch UID
   hooks:
```

Comment:
Where is this moved to?

Reply:
This has been removed. Due to this only openebs-upgrade was not starting as it was trying to find the csidrivers resource.

Reply:
This is basically a configuration which tells metac that this is a particular resource which it has to watch i.e., start watchers for it but once it has been given as config, it is assumed by metac that this will always be present in the cluster but in this case, it can be missing also for k8s versions prior to 1.14.0.